packages/experimental/ActivityIndicator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@fluentui-react-native/experimental-activity-indicator",
+  "version": "0.1.0",
+  "description": "A cross-platform Fluent Activity Indicator component",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "verify-api": "fluentui-scripts verify-api-extractor",
+    "update-api": "fluentui-scripts update-api-extractor"
```

Comment:
I think we also want depcheck here?

Reply:
I'm not sure what that means or does but from looking at other packages, it seems that some have it (Avatar and Button) and some don't (Shimmer and Icon).

Reply:
depcheck is a script that makes sure that the dependencies we specify are accurate. It will error if we have an unused dependency in that package, or use a dependency that isn't listed. We should probably add it, then make a followup bug to add it to the other packages missing it, as it's a pretty important check.